### ISSUE TITLE
onPrint click the pdf behind the overlay is jumping to left 

### ISSUE THREAD

**Issue by dheerajm91** [stephanrauh/ngx-extended-pdf-viewer#478]
Hi,

I was using the 4.0.3 version in my app till today and i moved to the latest 5.0 and noticed that the print functionality was not working like before. Please help me to fix this as this is very critical for me.

Issue: 
When i click on print button, i see the print preview window.
I can see the actual pdf jumped to the left of the screen behind the print preview window as well.
If i put back to the 4.0.3 version it works as expected. The problem is when i am using the 5.0 or above version i see this issue. I have a presentation in 2 days please help me to resolve this as soon as possible and appreciate your good work.

Please find the attached screenshots and video as well.

On initial load before click print button:

<img width="1658" alt="Screen Shot 2020-09-27 at 11 37 05 PM" src="https://user-images.githubusercontent.com/25700737/94398832-2ca9c380-011b-11eb-8a61-4cad8489b42c.png">

[Issue - Observe behind the print window, PDF jumped] **After clicking on print button:**

<img width="1674" alt="Screen Shot 2020-09-27 at 11 37 26 PM" src="https://user-images.githubusercontent.com/25700737/94398952-624eac80-011b-11eb-972c-df3b7487cab9.png">

This is not any CSS issue on my end. I also took a blank project and i was able to reproduce this issue and if i put back to 4.0.3 it works fine, please consider this as a bug and resolve this.

Thank you

**Comment by stephanrauh** [comment]
I don't see the effect on https://pdfviewer.net. Can you send me a reproducer, please? You've mentioned a blank project, so maybe you've got already a reproducer.

(The only bug I observed is Safari want to print the page instead of the PDF file when I try to print a second time).

**Comment by stephanrauh** [comment]
@dheerajm91 I'm trying to guess your environment. You're using Chrome on a Mac, correct?

**Comment by dheerajm91** [comment]
> I don't see the effect on https://pdfviewer.net. Can you send me a reproducer, please? You've mentioned a blank project, so maybe you've got already a reproducer.
> 
> (The only bug I observed is Safari wants to print the page instead of the PDF file when I try to print a second time).



> @dheerajm91 I'm trying to guess your environment. You're using Chrome on a Mac, correct?

Yes I am using Chrome on Mac.

**Comment by stephanrauh** [comment]
@dheerajm91 Can you share the code of the blank project with me? I can't help you as long as I can't reproduce the bug - and I believe the blank project isn't completely blank. :)

**Comment by dheerajm91** [comment]
> @dheerajm91 Can you share the code of the blank project with me? I can't help you as long as I can't reproduce the bug - and I believe the blank project isn't completely blank. :)

Sure i will share it now.

**Comment by dheerajm91** [comment]
Steps to reproduce:
Take a fresh project - ng new bug-tester

Please find the project Zipped without node modules:
Please do install the node modules once you take this project:
[bug-tester.zip](https://github.com/stephanrauh/ngx-extended-pdf-viewer/files/5293388/bug-tester.zip)

I will still paste the component / HTML code:

Component:
```
import { Component } from '@angular/core';
import { NgxExtendedPdfViewerService } from 'ngx-extended-pdf-viewer';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent {
  constructor(private printService: NgxExtendedPdfViewerService) {}
  title = 'bug-tester';
  url = 'https://vadimdez.github.io/ng2-pdf-viewer/assets/pdf-test.pdf';

  onPrint(): void {
    this.printService.print();
  }
}
```

HTML Code:

```
<button class="btn" (click)="onPrint()">Print</button><br>
<ngx-extended-pdf-viewer
[src]="url"
[useBrowserLocale]="true"
height="calc(100vh - 475px)"
zoom="50%"
[showSidebarButton]="false"
[showFindButton]="false"
[showPagingButtons]="false"
[showZoomButtons]="false"
[showPresentationModeButton]="false"
[showOpenFileButton]="false"
[showPrintButton]="false"
[showDownloadButton]="false"
[filenameForDownload]="false"
[showBookmarkButton]="false"
[showSecondaryToolbarButton]="true"
[showRotateButton]="false"
[showHandToolButton]="false"
[showScrollingButton]="false"
[showSpreadButton]="false"
[showPropertiesButton]="false">
</ngx-extended-pdf-viewer>

```

imported the NgxExtendedPdfViewerModule in the app module.


Reproducible screen shot:
Before print button click:
![image](https://user-images.githubusercontent.com/25700737/94453900-183ee880-0166-11eb-8ed4-aeb41e41ad26.png)

After Print button click: [Issue: Observe PDF position behind the print window]
![image](https://user-images.githubusercontent.com/25700737/94454027-3dcbf200-0166-11eb-9b83-9817f4c0efb5.png)

Please let me know if you need any other additional info?

Thank you

**Comment by stephanrauh** [comment]
I've managed to reproduce the bug using your reproducer. Now I'm puzzled! :)

**Comment by stephanrauh** [comment]
Stupid me... encapsulating CSS code (#435) is good and well, but it shouldn't encapsulate the code for printing!

The bug fix has landed with version 5.0.0-beta.1. Thanks again for providing the reproducer!

Best regards,
Stephan

**Comment by dheerajm91** [comment]
@stephanrauh  Thank you so much for the quick fix.
